library/core/src/cell.rs

```diff
@@ -1609,28 +1610,29 @@ impl<T: ?Sized + fmt::Display> fmt::Display for RefMut<'_, T> {
 /// }
 /// ```
 ///
-///   - Exclusive access to an `UnsafeCell<T>` ⇒ exclusive access to its `T`:
+/// The following example showcases the fact that exclusive access to an `UnsafeCell<T>`
+/// implies exclusive access to its `T`:
```

Comment:
Now the first example, the one right after `# Examples`, sticks out because it has no text above it to say what it is about. Can you add a brief sentence there?